CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ All notable changes to this project will be documented in this file.
 
 ## [[NEXT]](https://github.com/iExecBlockchainComputing/iexec-blockchain-adapter-api/releases/tag/vNEXT) 2024
 
+### New Features
```

Comment:
I think I would rather add this to the **Quality** section as the business code (`main` source set) is not modified.
What do you think ?

Reply:
I was not sure of my choice so **Quality** is good to me !